fluent-bundle/src/args.rs

```diff
@@ -118,3 +117,20 @@ impl<'args> IntoIterator for FluentArgs<'args> {
         self.0.into_iter()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn replace_existing_argument() {
+        let mut args = FluentArgs::new();
+        args.set("variable", "first value");
```

Comment:
suggestion: Could you add another set of assertions like so, just to fully demonstrate the behavior is correct here?

```suggestion
        args.set("variable", "first value");
        assert_eq!(args.0.len(), 1);
        assert_eq!(
            args.get("variable").unwrap(),
            &FluentValue::from("first value")
        );
```

Reply:
I've changed the test a little bit to add more assertions and bit less abstract, is that enough? Or did you have something else in mind? Don't hesitate to let me know 👍